neo4j/_async/work/transaction.py

```diff
@@ -131,6 +134,43 @@ async def run(self, query, parameters=None, **kwparameters):
 
         return result
 
+    async def query(self, query, parameters=None, **kwparameters):
+        """ Run a Cypher query within the context of this transaction.
+
+        Cypher is typically expressed as a query template plus a
+        set of named parameters. In Python, parameters may be expressed
+        through a dictionary of parameters, through individual parameter
+        arguments, or as a mixture of both. For example, the `run`
+        queries below are all equivalent::
+
+            >>> query = "CREATE (a:Person { name: $name, age: $age })"
+            >>> query_result = await tx.query(query, {"name": "Alice", "age": 33})
+            >>> query_result = await tx.query(query, {"name": "Alice"}, age=33)
+            >>> query_result = await tx.query(query, name="Alice", age=33)
+
+        Parameter values can be of any type supported by the Neo4j type
+        system. In Python, this includes :class:`bool`, :class:`int`,
+        :class:`str`, :class:`list` and :class:`dict`. Note however that
+        :class:`list` properties must be homogenous.
+
+        :param query: cypher query
+        :type query: str
+        :param parameters: dictionary of parameters
+        :type parameters: dict
+        :param kwparameters: additional keyword parameters
+
+        :returns: a new :class:`neo4j.QueryResult` object
```

Comment:
The type is already documented below :upside_down_face: 
```suggestion
        :returns: the result of the query
```